controllers/pullRequestsController.js

```diff
@@ -0,0 +1,47 @@
+const logger = require('../utils/logger')
+const fetch = require('../lib/fetch')
+const config = require('../config')
```

Comment:
Use the library `'config'`, do not read from the file. Refer in other files.

Reply:
This should solve the failing CI problem

Reply:
Cool